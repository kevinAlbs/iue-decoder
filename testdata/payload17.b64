ERI0VngSNJh2EjQSNFZ4kBICBQAAAAIAAAABAAAAZqAy0bSOXpYYlpKQxCYMoJiPnn1LGwTohBSMPC1+G93bEjHAlofOenlBB9n92xusO/oky0LQKXwWrUQGlgeGvr+a6dcJziqJgr94tT5e5FYQdipids631+ctrY6L6zV9NLwbMs5kBPPjnJf9uGsHfXDIaJb6Mp+vMCmhQBL+S3Q9ImfZi4iOkAbGgwOcGhNHuvGOUPAnBLFGv8AYz0GlV1KBX1PVeVxmrQ7S8r3n9kcdoLfv/KT+ITaV8qFqub4TZnj/kp9PB3sAlngv1Kw8KWu/Kv8lIKylWFz6jI0U1IHFcB6LLGhLJVkR18PhoPGpPXW4mU9vdiEJG9Bi2GxDmsDQg2sw4MnSTtgNIzoJ70QfPh8ItSXJ0mgrI+QyvG9YXeTAqznIfkyws6Y+Xqh4wQ5wWsxMWRJvhiU2zNeWg9MYubWD8j2lcTJE5M/57I+6WhKtu7dRP/AwRYak9M++MBSkIZF5p4YYz4iQgCYrS5Z4b7c9Syqyw5q1oJNNmzEGUU2UwHKWvmy9mvU7Re5FB8kjzcIwd5p1wzdy33uISqiY056xvhCODyhdzSqi8OKd2HoEh22275uI5DOOelby31JnznTSreIHj8XEwtLTQem/qSGRLido/+f4H9AqP8Jqd0rfMyK6EaedeL+Cd5yiiEVjtLLX2whPoSnOmztiiCEBFUrqzKPyYVhzzySnLPOaeRpDQix5xlCXEtzNqhYyMiot/wCkSM6wuYkYzRJ0/ug6fDPBpA/xdpP3jXhbM0vc