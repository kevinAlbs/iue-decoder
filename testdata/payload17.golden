BlobSubtype=17 (FLE2IndexedTextEncryptedValue)
key_uuid=12345678123498761234123456789012
OriginalBsonType=2 (string)
EdgeCount=5
SubstrTagCount=5
SuffixTagCount=5
ServerEncryptedValue=
Exact.encryptedCount=66A032D1B48E5E9618969290C4260CA0988F9E7D4B1B04E884148C3C2D7E1BDD
Exact.tag=DB1231C09687CE7A794107D9FDDB1BAC3BFA24CB42D0297C16AD4406960786BE
Exact.encryptedZeros=BF9AE9D709CE2A8982BF78B53E5EE45610762A6276CEB7D7E72DAD8E8BEB357D
Substr.encryptedCount[0]=34BC1B32CE6404F3E39C97FDB86B077D70C86896FA329FAF3029A14012FE4B74
Substr.tag[0]=3D2267D98B888E9006C683039C1A1347BAF18E50F02704B146BFC018CF41A557
Substr.encryptedZeros[0]=52815F53D5795C66AD0ED2F2BDE7F6471DA0B7EFFCA4FE213695F2A16AB9BE13
Substr.encryptedCount[1]=6678FF929F4F077B0096782FD4AC3C296BBF2AFF2520ACA5585CFA8C8D14D481
Substr.tag[1]=C5701E8B2C684B255911D7C3E1A0F1A93D75B8994F6F7621091BD062D86C439A
Substr.encryptedZeros[1]=C0D0836B30E0C9D24ED80D233A09EF441F3E1F08B525C9D2682B23E432BC6F58
Suffix.encryptedCount[0]=5DE4C0AB39C87E4CB0B3A63E5EA878C10E705ACC4C59126F862536CCD79683D3
Suffix.tag[0]=18B9B583F23DA5713244E4CFF9EC8FBA5A12ADBBB7513FF0304586A4F4CFBE30
Suffix.encryptedZeros[0]=14A4219179A78618CF889080262B4B96786FB73D4B2AB2C39AB5A0934D9B3106
Suffix.encryptedCount[1]=514D94C07296BE6CBD9AF53B45EE4507C923CDC230779A75C33772DF7B884AA8
Suffix.tag[1]=98D39EB1BE108E0F285DCD2AA2F0E29DD87A04876DB6EF9B88E4338E7A56F2DF
Suffix.encryptedZeros[1]=5267CE74D2ADE2078FC5C4C2D2D341E9BFA921912E2768FFE7F81FD02A3FC26A
Prefix.encryptedCount[0]=774ADF3322BA11A79D78BF82779CA2884563B4B2D7DB084FA129CE9B3B628821
Prefix.tag[0]=01154AEACCA3F2615873CF24A72CF39A791A43422C79C6509712DCCDAA163232
Prefix.encryptedZeros[0]=2A2DFF00A448CEB0B98918CD1274FEE83A7C33C1A40FF17693F78D785B334BDC
